hack/make-rules/test.sh

```diff
@@ -365,16 +358,7 @@ checkFDs() {
 
 checkFDs
 
-
-# Convert the CSVs to arrays.
-IFS=';' read -r -a apiVersions <<< "${KUBE_TEST_API_VERSIONS}"
-apiVersionsCount=${#apiVersions[@]}
-for (( i=0; i<apiVersionsCount; i++ )); do
-  apiVersion=${apiVersions[i]}
-  echo "Running tests for APIVersion: ${apiVersion}"
-  # KUBE_TEST_API sets the version of each group to be tested.
-  KUBE_TEST_API="${apiVersion}" runTests "$@"
-done
+runTests
```

Comment:
This change looks kind of risky, how can we be sure that we're running the same tests we were before?

Shouldn't you be passing something to this? in the loop it takes an argument...

Reply:
+1, AFAICT this should continue to take "$@"